Story-Chief/frontend-test-josepmiquel
# Feature/searchbar

## Description

I added a new feature to search and filter events by text.


## Closes issue(s)



## Testing Instructions
1. Write some text in the search bar
2. Check if the events are filtered by text (including title and description)





## Screenshots
